client/src/i18n/en/patient_records.json

```diff
@@ -58,6 +58,18 @@
       "IS_NOT_NEW_CASE":"Old Case",
       "IS_REFERED":"Is the patient a referral ?"
     },
+    "REPORT":{
+      "VISITS":"Patients Visits Report",
+      "DESCRIPTION":"This report shows visits stats by service",
+      "NEW_CASES":"New Cases",
+      "OLD_CASES":"Old Cases",
+      "HOSPITALIZED":"Hospitalized",
+      "AMBULATORY":"Ambulatory",
+      "PREGNANT":"Pregnants",
+      "REFERRED":"Referred",
+      "INSIDE_HEALTH_ZONE":"Inside Health Zone",
```

Comment:
While this is technically correct, a cleaner way to say this in English is "Within Health Zone".  Just FYI :)